_posts/2017-04-30-powershell.md

```diff
@@ -1,6 +1,10 @@
 ---
 layout: post
 title: "Powershell"
-code: 'PS C:\>0.1 + 0.2'
-result: '0.3'
+code: 
```

Comment:
Unfortunately, this wont display the way you want. Lists of code are used to show multiple examples, so they will all be separated with "And". If you change your code sample to the following, I'll merge it.

```
code: |-
  $a = 0.1
  $b = 0.2
  write-host ($a+$b)
```